packages/fxa-payments-server/server/lib/amplitude.test.js

```diff
@@ -210,6 +182,16 @@ describe('lib/amplitude', () => {
       expect(log.info).toHaveBeenCalledTimes(1);
     });
   });
+  describe('respects do not track', () => {
```

Comment:
nit: Check the converse. i.e. that dnt isn't accidentally called when the dnt header isn't set.